assignment.md

```diff
@@ -12,46 +12,86 @@ If the version control concepts or `git` commands are still confusing to you, co
+* df90802 updated syllabus with presidents day
+* e0029df place holder schedule
+* 7462142 update lecture link for week 3
+* 8fced37 assignment 2 updated
+* 2a2bfd5 version control chapter
+* 13e5722 minor improvements to instructions
+* dd4719c fixed assignment 1 link
+* 0392554 fixed syllabus
+* 48e689c updated syllabus with MLK day
+* e96eba0 added dave's slides link
+* fbb9e8d fixed typo
+* c152f3c added ch2 assignment
+* cd40b26 fixed typo in assignment 1
+* b3455f6 fixed missing link
+* 7cd4587 update links to reflect erlative links
+* fb12bef jekyll config additions for relative links
+* 7ed94e9 dash fix
+* d6f3ec1 fixed link error
+* 2c05357 update links
         ```
 
     - b. Who was the last person to modify `README.md`? (Hint: use `git log` with an argument)
         ```bash
         # git command and output here
+        git log README.md
+commit 00e8ae51a08a0f3754ab228f62b51651595c4426
+Author: Xihe Xie <axiezai@gmail.com>
+Date:   Mon Dec 28 16:01:51 2020 -0500
+
         ```
     
     - c. What was the commit message associated with the last modification to the `-Week 2` line of `index.md` file? (Hint: use `git blame` and `git show`, check out their help pages if you are not sure what to do)
 
+    The commit message was "added ch2 assignment."
+
  5. In the course website repository (or another repository you want to clone from GitHub), modify one of its existing files, do not `git add` or `git commit` yet:
      - a. What happens when you do `git stash`, be specific about what happens to your uncommitted changes after `stash`.?
-       > Answer here
+       > It saves uncommitted changes in the stash log and removes them from the current directory. 
      - b. What do you see when you run `git log --all --oneline` after stashing your modification?
-       > Answer here
+       > My stash is shown on top of the log.
      - c. Look up what `git stash drop` and `git stash pop` does, either Google or use `--help`. Run `git stash drop` to undo what you did with `stash`, in what scenario might `stash`-ing, `pop`-ing, and `drop`-ing be useful?
-       > Answer here
+       > 'git stash drop' drops a single stash entry. These three commands allow you to save changes and try out different options and can bring previous changes back if you change your mind. You can also get rid of the changes by dropping the stash. 
 
  6. One common mistake when learning `git` is to commit large files that should not be managed by `git` or adding sensitive information like security keypairs for Amazon Cloud Services. Navigate back to your assignment repository and try adding a random text file to the repository (use `touch`), making some modifications and commits to your repository, and deleting that file with `rm`. But deleting with `rm` will not delete that file's recorded git history, therefore we need to do a little more. You may want to look at [this](https://help.github.com/articles/removing-sensitive-data-from-a-repository/))
```

Comment:
-10%

The point of this question is to have you remove committed content. You submitted 1 c 

I was expecting the following:

 - One commit where you create a new file that has a few lines of text (fake sensitive info)
 - Another commit where you delete the file. (At this point, your commit history still has the sensitive info saved as a commit/version)
 - Another commit where you use `filter-branch` and `git rm` to delete the sensitive text. 

None of these were found.